src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -59,6 +67,62 @@ internal void InitializeLicense(string licenseText)
             license = foundLicense;
         }
 
+        static bool FirstTimeUser()
+        {
+            // Check for the presence of HKCU:Software/NServiceBus
+            using (var regNsbRoot = Registry.CurrentUser.OpenSubKey(@"Software\NServiceBus"))
+            {
+                if (regNsbRoot == null)
+                {
+                    //Check for the presence of HKCU:SOFTWARE\ParticularSoftware
+                    using (var regParticularRoot = Registry.CurrentUser.OpenSubKey(@"Software\ParticularSoftware"))
+                    {
+                        if (regParticularRoot == null)
+                        {
+                            return true;
+                        }
+                    }
+                }
+            }
+            return false;
+        }
+
+        /// <summary>
+        /// Adding the async void to explicitly have a fire and forget model for invoking the web api.
+        /// </summary>
+        async void TrackFirstTimeUsageEvent()
+        {
+            // Get the current version of NServiceBus that's being used
+            var version = FileVersionRetriever.GetFileVersion(GetType());
+
+            // Report first time usage metric
+            Logger.InfoFormat("Reporting first time usage and version information to www.particular.net.  This call does not collect any personal information. For more details, see the License Agreement and the Privacy Policy available here: http://particular.net/licenseagreement.  This call will NOT be executed in production servers. It is executed only once for the first time, only when run in an interactive debugging mode.");
```

Comment:
Why are there two spaces between some sentences and one space in others? I'd prefer one space for each.

Also, this sentence:
"It is executed only once for the first time, only when run in an interactive debugging mode."

is awkward. At the very least, it needs an "and before the "only. But, the first part is unclear.